src/Network/X509/SelfSigned.hs

```diff
@@ -156,6 +158,23 @@ import System.X509 (getSystemCertificateStore)
 -- internal modules
 
 import Chainweb.Utils
+    ( HasTextRepresentation(..)
```

Comment:
this seems excessive

Reply:
I did this, because 'Network.X509.SelfSigned' is a candidate for being moved to a package on its own. `Chainweb.Utils` is the only internal dependency and I though I would be a good idea to keep track on what exactly the module depends on. I left it in the code so that I don't have to come up with the list again next time I work on this module.

Reply:
Fair enough